Longman Dictionary extension

veggiecan0419 opened the pull request:
Ever needed to get the definitions of words in your projects? 
If so, here it is.

GarboMuffin commented:
Icon works poorly in dark mode

![image](https://github.com/TurboWarp/extensions/assets/33787854/8f1d1b9b-758a-4ea8-aab7-0a17e9ba4b36)

You will need to handle fetch() errors and return a readable error message instead of letting it return undefined

veggiecan0419 commented:
**Update details**
- icon now works in dark and light mode
- fixed the _fetch()_ errors
- renamed 'definition of...' to 'primary definition of...'
- added 'all definitions of [word] in the Longman Dictionary '
- added credits to veggiecan0419
